toolkit/README.md

```diff
@@ -0,0 +1,81 @@
+- [Supporting Documentation](#supporting-documentation)
+- [Prerequisite](#prerequisite)
+  - [Install Dependencies](#install-dependencies)
+- [Write Codeblocks](#write-codeblocks)
+  - [Write Codeblocks Usage](#write-codeblocks-usage)
+- [Check CLI Output](#check-cli-output)
+  - [Check CLI Output Usage](#check-cli-output-usage)
+
+
+# Supporting Documentation
+
+If you are using this toolkit to add/edit the current documentation, please read the [contributing documentation](https://developer.fermyon.com/spin/v2/contributing-docs) page (and perhaps leave it open for reference when forking, cloning, committing, etc.).
+
+# Prerequisite
+
+Ensure that you have installed Python 3.10 or later on your system. You can check your Python version by running:
+
+```bash
+python3 --version
+```
+
+## Install Dependencies
+
+Change into an arbitrary working directory so as not to add versioned files to this repository. The files created by this toolkit are auxiliary and are manually inserted into the versioned documentation files. We will use the system's home directory for demonstration purposes (and assume that you have the developer repo cloned into your home directory i.e. `~/developer`):
+
+```bash
+cd ~
+```
+
+Set up and activate a Python virtual environment and then install the dependencies:
+
+```bash
+# Create venv
+python3 -m venv venv-dir
+# Activate venv
+source venv-dir/bin/activate
+# Install requests library
+pip3 install requests
+```
+
+# Write Codeblocks
+
+The `update_spin_cli_reference.py` script will fetch the current [Spin CLI Reference](https://developer.fermyon.com/spin/v2/cli-reference) document from the web and write a fresh copy of the source markdown file (a new file called `updated_markdown.md` with the new version of code blocks added). The fresh markdown file, generated by this script, is the basis of a PR that contains the CLI commands for the new version of Spin.
+
+## Write Codeblocks Usage
+
+Run the script and pass in the two versions of Spin.
+
+**Note**: Use two number characters `vX.X` only i.e. `v2.6`:
+
+```bash
+python3 developer/toolkit/update_spin_cli_reference.py v2.6 v2.7
+```
+
+# Check CLI Output
+
+The `script_to_check_cli_outputs.sh` script will tell you what the changes are (between two different Spin versions) the script is designed to quickly kick off the edit, review, approve, and merge process at the time of release.
+
+## Check CLI Output Usage
+
+Run the script using the `vX.X.X` format. Please note, the Spin versions you use must exist as a Spin release in the Spin GitHub repo.
+
+**Note**: Use three number characters `vX.X.X` i.e. `v2.6.0`:
+
+```bash
+# Assuming this repo is cloned to your home directory (as per the first example also)
+cd ~
+cd developer/toolkit
+# Make the script executable
+sudo chmod a+x script_to_check_cli_outputs.sh
```

Comment:
We should be able to get rid of this step if you run `chmod +x script_to_check_cli_outputs.sh` and commit the result.